src/core/evaluator.js

```diff
@@ -459,14 +459,14 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         return;
       }
 
-      var useNativeImageDecoder = !this.options.disableNativeImageDecoder;
+      var useNativeImageDecoder = this.options.nativeImageDecoderSupport;
```

Comment:
This variable name is now misleading, please change it to `var nativeImageDecoderSupport` instead.